PaymentHighway/Core/Environment.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Properties.swift
+//  PaymentHighway
+//
+//  Copyright © 2018 Payment Highway Oy. All rights reserved.
+//
+
+import Foundation
+
+public enum Environment: ServerType {
+    
+    #if DEBUG
+    public static var current: Environment = .sandbox
```

Comment:
I think this should just default to `.sandbox` always and let the consumer define when they want to use `.production` explicitly?

Reply:
thanks! Agree that might be confusing.
We have been discussing here about have the default baseurl with sandbox or not and conclusion has been that customer should set the base url explicitly always. So i will do the changes  to make this works in this way.